### PR TITLE
Extracted configtxgen and required dependencies from fabric-sdk-go.

### PR DESCRIPTION
**Pull request by alikic**
Signed-off-by: Aleksandar Likic <aleksandar.likic@securekey.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+# Fabric tool docker tags at code levels
+FABRIC_TOOLS_STABLE_TAG     = $(FABRIC_ARCH)-$(FABRIC_STABLE_VERSION)
+FABRIC_TOOLS_PREV_TAG       = $(FABRIC_ARCH)-$(FABRIC_PREV_VERSION)
+FABRIC_TOOLS_PRERELEASE_TAG = $(FABRIC_ARCH)-$(FABRIC_PRERELEASE_VERSION)
+FABRIC_TOOLS_DEVSTABLE_TAG  := stable
+
+# Detect CI
+# TODO introduce nightly and adjust verify
+ifdef JENKINS_URL
```
Comment (troyronda) [inline Makefile]: You will need a CI environment variable. E.g., in fabric-sdk-go JENKINS_URL was defined as a quick fix. https://github.com/hyperledger/fabric-sdk-go/blob/master/ci/azure-pipelines.yml#L42

**Comment by troyronda** [comment]
Can we get rid of third_party?

**Comment by alikic** [comment]
The plan is to move third_party stuff to top level as these should be first class packages now.

**Comment by troyronda** [comment]
> The plan is to move third_party stuff to top level as these should be first class packages now.

I guess I meant: it would be nice not to expose **copied** packages at all :).

**Comment by alikic** [comment]
It is true, the ones which we don't want to expose should be internal. But it's not all third_party.

**Comment by alikic** [comment]
I moved all third_party to internal. Now we need integration tests to tell us which internal functions we need to expose for desired use cases.

**Comment by kopaygorodsky** [comment]
I see you made whole encoder pkg internal, any chance it could be exposed?  I think any function that works with genesis config should be exposed, need some way to convert Profile to ConfigGroup

**Comment by alikic** [comment]
Closing, moving to #5

**Pull request closed without merging**